documentation/modules/creating-vddk-image.adoc

```diff
@@ -8,7 +8,7 @@
 
 {the} {project-first} uses the VMware Virtual Disk Development Kit (VDDK) SDK to transfer virtual disks from VMware vSphere.
 
-You must download the VMware Virtual Disk Development Kit (VDDK), build a VDDK image, and push the VDDK image to your image registry. Later, you will add the VDDK image to the `HyperConverged` custom resource (CR).
+You must download the VMware Virtual Disk Development Kit (VDDK), build a VDDK image, and push the VDDK image to your image registry.
```

Comment:
I think it will be good to add here that later this VDDK init image path will be added as part of adding VMware provider.

and further in the document, in Add provider via GUI or CLI reference to this section.